.vscode/settings.json

```diff
@@ -1,6 +1,25 @@
 {
+  "C_Cpp.autoAddFileAssociations": false,
   "files.associations": {
-    "*.overlay": "dts",
-    "*.keymap": "dts"
```

Comment:
ZMK Tools now requires that .keymap files *not* be overridden so it can handle them as keymap files and not DT files.